sphinx/source/docs/user_guide/examples/styling_math_text_tick_labels.py

```diff
@@ -0,0 +1,23 @@
+from numpy import arange
+
+from bokeh.models import FixedTicker
+from bokeh.plotting import figure, show
+
+x = arange(1, 4.5, 0.25)
+y = 1 / x
+
+plot = figure(height=200)
+plot.circle(x, y, fill_color="blue", size=5)
+
+plot.xaxis.axis_label = "Resistance"
+plot.xaxis.ticker = FixedTicker(ticks=[1,2,3,4])
```

Comment:
Unless you specifically wanting to be verbose, this can be abbreviated
```
plot.xaxis.ticker = [1,2,3,4]
```
(an then the import can be skipped as well)